spec/factories/lists.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :list do
+    title "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.